conf/distro/tb-distro.conf

```diff
@@ -0,0 +1,51 @@
+require conf/distro/poky.conf
+
+DISTRO = "tb-distro"
+
+DISTRO_NAME = "Reference Yocto distro for PC Engines hardware"
```

Comment:
`"Reference Yocto distro for demonstration purposes of TrenchBoot project"`

Reply:
Why not Zarhus?

Reply:
Because TrenchBoot :thinking:

Reply:
I have no idea what this answer means. Everything what 3mdeb provide regarding `Yocto distro` is Zarhus.

Reply:
Let's use `Zarhus distro for demonstration purposes of TrenchBoot project` then ?

Reply:
Changed `DISTRO_NAME` in [17e3553](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/17e35533adf6a5d6d3b62703f50865732f5a456d)